study-builder/studies/rarex/quality-of-life-parental.conf

```diff
@@ -1,1443 +0,0 @@
-{
-  include required("../snippets/activity-general-form.conf"),
-  "studyGuid": "RAREX",
-  "activityCode": "PARENTAL_QUALITY_OF_LIFE",
```

Comment:
This is split into `quality-of-life-child.conf` and `quality-of-life-patient.conf`.